QualityControl/lib/dtos/LayoutDto.js

```diff
@@ -84,6 +84,7 @@ export const LayoutDto = Joi.object({
 export const LayoutsGetDto = Joi.object({
   owner_id: Joi.number().integer().optional(),
   name: Joi.string().optional(),
+  object_path: Joi.string().pattern(/^[A-Za-z0-9_/]+$/).optional().max(100).label('Object path'),
```

Comment:
The DTO is not correct. Objects can also contain for example the charcater `-` which I have seen that you added tests for being invalid. 
If you have a look through our production instance you can see what characters we should allow or not